erroranalysis/erroranalysis/_internal/surrogate_error_tree.py

```diff
@@ -429,13 +537,35 @@ def get_regression_metric_data(df):
 
 
 def get_classification_metric_data(df):
+    """Compute classification metric data from a dataframe.
+
+    :param df: dataframe
+    :type df: pandas.DataFrame
+    :return: pred_y, true_y, error
+    :rtype: numpy.array, numpy.array, int
```

Comment:
Is this numpy.ndarray or numpy.array?

import numpy as np
np.ndarray
<class 'numpy.ndarray'>
np.array

Seems like numpy.array is built-in function array.

Reply:
done, replaced numpy.array with numpy.ndarray everywhere